examples/webgl_loader_draco.html

```diff
@@ -38,12 +38,14 @@
 		<a href="https://github.com/google/draco" target="_blank" rel="noopener">DRACO</a> loader
 	</div>
 	<script src="../build/three.js"></script>
-	<script src="js/loaders/draco/draco_decoder.js"></script>
 	<script src="js/loaders/draco/DRACOLoader.js"></script>
+	<script src="js/loaders/draco/geometry_helper.js"></script>
```

Comment:
Why not include this inside `DRACOLoader.js`?

Reply:
I found out this was not used in this example. So removed the file and the line.